bigchaindb/processes.py

```diff
@@ -11,9 +11,10 @@
 BANNER = """
 ****************************************************************************
 *                                                                          *
-*   Initialization complete. BigchainDB is ready and waiting for events.   *
-*   You can send events through the API documented at:                     *
-*    - http://docs.bigchaindb.apiary.io/                                   *
+*   Initialization complete. BigchainDB Server is ready and waiting.       *
+*   You can send HTTP requests via the HTTP API documented in the          *
+*   BigchainDB Server docs at:                                             *
+*    https://docs.bigchaindb.com/projects/server/en/latest/index.html      *
```

Comment:
What do you think of giving a short URL from a URL-shortening service like Bitly? For example:  http://bit.ly/bigchaindb-http